core/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -734,9 +734,20 @@ public BytesReference ext() {
         return ext;
     }
 
+    /**
+     * Create a new SearchSourceBuilder with attributes set by an xContent.
+     */
     public SearchSourceBuilder fromXContent(XContentParser parser, QueryParseContext context, AggregatorParsers aggParsers)
             throws IOException {
         SearchSourceBuilder builder = new SearchSourceBuilder();
+        builder.parseXContent(parser, context, aggParsers);
```

Comment:
> add a small comment to the merge commit

I'll try and remember the comment in the merge commit but there isn't a good place in my workflow for it :(. I'll absolutely add a comment to parseXContent as part of the PR and merge it in manually.